src/test/java/uk/gov/companieshouse/api/accounts/controller/FilingControllerTest.java

```diff
@@ -0,0 +1,95 @@
+package uk.gov.companieshouse.api.accounts.controller;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import uk.gov.companieshouse.api.accounts.model.entity.CompanyAccountEntity;
+import uk.gov.companieshouse.api.accounts.model.filing.Filing;
+import uk.gov.companieshouse.api.accounts.service.FilingService;
+import uk.gov.companieshouse.api.accounts.transaction.Transaction;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(Lifecycle.PER_CLASS)
+public class FilingControllerTest {
+
+    private static final String TRANSACTION_ID = "1234561-1234561-1234561";
+    private static final String ACCOUNTS_ID = "1234561";
+    private ResponseEntity response;
+
+    @Mock
+    private FilingService filingServiceMock;
+    @Mock
+    private Transaction transactionMock;
+    @Mock
+    private CompanyAccountEntity companyAccountEntityMock;
+    @Mock
+    private HttpSession httpSessionMock;
+    @Mock
+    private HttpServletRequest httpServletRequestMock;
+    @InjectMocks
+    private FilingController filingController;
+
+    @BeforeEach
+    public void setUp() {
+        when(httpServletRequestMock.getAttribute(anyString()))
```

Comment:
Can this be split into 2 `when` conditions with specific attribute names? This will make the tests less brittle.